src/plugins/intel_gpu/src/graph/network.cpp

```diff
@@ -1224,43 +1241,121 @@ void network::execute_impl(const std::vector<event::ptr>& events) {
     };
 
     for (auto& inst : _exec_order) {
+        // Load binary dump for input layers
+        GPU_DEBUG_IF(!debug_config->load_layers_raw_dump.empty()) {
+            const std::string layer_name = inst->id();
+            GPU_DEBUG_IF(debug_config->is_layer_for_loading(layer_name)) {
+                auto files = debug_config->get_loading_filenames(layer_name);
+                if (inst->is_input()) {
+                    // Binary dump for input layer : only one output exists or index(dstN) exists
+                    auto dump_file = debug_config->get_matched_from_filelist(files, "_dst0__");
```

Comment:
Dumping and loading any layer do for multiple input/output tensor. So it needs to identify index of src or index of dst. Do you have other idea of matching index?

Reply:
+ Just for loading input layer, if there is only one tensor input, then it does NOT require the pattern 'dstN'.

Reply:
* I think this behavior is too implicit. What about guessing the input port from the input index?
* I think the behavior is a bit complicated because sometime we use src(for non-parameters), sometimes we use dst(for parameters). What about just supporting dst only?

Reply:
conclusion is to improve error message